packs/feats/spell-trickster-dedication.json

```diff
@@ -31,7 +31,187 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: The Grand Bazaar"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.SpellTrickster.Label",
+                "option": "spell-trickster",
+                "placement": "spellcasting",
+                "suboptions": [
+                    {
+                        "label": "PF2E.SpecificRule.SpellTrickster.AgileHand.Label",
+                        "predicate": [
```

Comment:
Do any spells have multiple tricks possible? If so then we should use this approach but wait for us to have dynamic suboptions so 3pp can be supported. If not just a single toggle to use your trick.

Reply:
You can only use one modification at a time per the dedication's rules
> You can only apply one modification to a spell at a time, even if you know more than one modification for that spell.

Reply:
I don't think we should be doing prereqs as RE stuff, no. But yeah maybe we stew on this until we have dynamic suboptions.

Reply:
We'll need that for amps too.